### ISSUE TITLE
NTR: Axon flare response absent after intradermal histamine injection (Autonomic)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7778]
New term request Axon flare response absent after intradermal histamine injection (segal_030124164958):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000322
- label: Absent axon flare response after intradermal histamine injection
- parents: [Abnormal peripheral nervous system physiology (HP:0032120)](https://hpo.jax.org/app/browse/term/HP:0032120)
- pmid: PMID:2456128
- def: The axon reflex (also known as the flare response) is a response that is stimulated by peripheral nerves of the body that travels away from the nerve cell body and branches to stimulate target organs. In the human skin, there are three different phenomena mediated by axon reflex mechanism; flare, sweating and pilomotion. Stroking or intradermal injection of histamine or substance P into the human skin pro- duces widespread vasodilatation (flare) and plasma extravasation (wheal). This term refers to the failure of intradermal hista injection to elicit flare.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7778
- synonyms:
